.gitignore

```diff
@@ -4,6 +4,7 @@ coverage.*
 tools
 build
 .vscode
+docs/scripts/generated_script.sh
```

Comment:
Who is generating this?  What's the relation to golang version?

Reply:
the docs-ecutable metascript